src/base/baseRepositoryService.ts

```diff
@@ -17,9 +17,10 @@ export default class BaseRepositoryService<Entity extends BaseEntity> {
    return this.repository.findOne(options);
   }
 
-  getPaginatedList(skip: number, itemsPerPage: number, matcher?: object): Promise<Entity[]> {
+  getPaginatedList(skip: number, itemsPerPage: number, matcher?: object, relations?: string[]): Promise<Entity[]> {
```

Comment:
Wouldn't it be better to get params as objects ```{skip: number, itemsPerPage: number, ...}``` so it will be more convenient to pass extra params
For example, to pass relations we need to pass matcher object first

Reply:
Done